packages/date-skeleton/src/get-date-formatter.test.ts

```diff
@@ -120,7 +120,8 @@ describe('Options', () => {
     expect([
       '2 Dhuʻl-Hijjah 1426',
       'Dhuʻl-Hijjah 2, 1426',
-      'Dhuʻl-Hijjah 2, 1426 AH'
+      'Dhuʻl-Hijjah 2, 1426 AH',
+      '2 Dhuʻl-Hijjah 1426 AH'
```

Comment:
Thank you, this looks like the right fix for this test. Would you be ok if I cherry-picked this commit separately from the rest of the PR?

Reply:
please do :)